### PR TITLE
Initial video support

### PR DESCRIPTION
**Pull request by cziegeler**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Test URLs:
- Before: https://main--audemarspiguet-careers--hlxsites.hlx.live/en/
- After: https://videos--audemarspiguet-careers--hlxsites.hlx.live/en/

### CODE REVIEW

**Review by hannessolo — CHANGES_REQUESTED** [review]
Approach looks good in general

Before merging could you:

* Improve the mobile experience - it doesn't need to use the carousel, the videos can just be laid out one after the other, or even hidden
* Make the video poster remain visible after the video player loads. For me all the videos become black after the player loads, see screenshot below
* Play around with the z-index of the video and nav bar, the videos go over the nav bar now


Screenshot 2023-05-05 at 14 35 25

**Comment by cziegeler** [comment]
The "black video box" seems to be a firefox issue, it works in chrome

File: scripts/delayed.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
 // eslint-disable-next-line import/no-cycle
-import { sampleRUM } from './lib-franklin.js';
+import { sampleRUM, loadScript } from './lib-franklin.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
 // add more delayed functionality here
+loadScript('/scripts/video.js', () => {
```
Comment (mhaack) [inline scripts/delayed.js:8]: We should load the video.js from the original host here, no need to copy this

Comment (cziegeler) [reply scripts/delayed.js]: Makes sense

File: .eslintignore [hunk 2]
```diff
@@ -1 +1,2 @@
-helix-importer-ui
\ No newline at end of file
+helix-importer-ui
+scripts/video.js
```
Comment (hannessolo) [inline .eslintignore:2]: I think this can be removed again since you deleted `scripts/video.js`

**Review by hannessolo — APPROVED** [review]
Look great, you can remove `scripts/video.js` from `.eslintignore` and then merge directly.

File: .eslintignore (hunk 2 above)
Comment (cziegeler) [reply .eslintignore]: Good catch, right

**Pull request merged**